privacy_gdpr/security/ir.model.access.csv

```diff
@@ -0,0 +1 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
```

Comment:
Remove this not empty file (no longer needed since changing wizard to TransientModel). Do not forget to also remove the corresponding line from the manifest file.